cmd/tx/tx_utils/tx_utils.go

```diff
@@ -15,13 +14,13 @@ func CheckTxStdOut(stdout bytes.Buffer) error {
 		return err
 	}
 
-	if strings.Contains(response.RawLog, "fail") || strings.Contains(response.RawLog, "error") {
+	if response.ExitCode != 0 {
 		return errors.New(response.RawLog)
 	}
-
 	return nil
 }
 
 type Response struct {
-	RawLog string `json:"raw_log"`
+	RawLog   string `json:"raw_log"`
+	ExitCode int    `json:"code"`
```

Comment:
It's not an ExitCode but a code that returns from the tx response. suggest changing the name as it's confusing.